CONTRIBUTING.md

```diff
@@ -65,3 +68,41 @@ using [a phrase such that github auto-closes the issue when merged to master](ht
 [issue-tracker]: https://github.com/CCPPETMR/SIRF/issues
 [pull-requests]: https://github.com/CCPPETMR/SIRF/pulls
 [googlegroup]: https://groups.google.com/forum/#!forum/ccp-petmr-codebot
+
+
+## Submodules
+
+> [...] A submodule is a repository embedded inside another repository. The
+> submodule has its own history [...]
+
+This project uses [submodules] to point to external libraries or frequently
```

Comment:
I guess remove the "external libraries" as we don't (yet).

Also "*in*frequently"